src/CMakeLists.txt

```diff
@@ -9,18 +9,22 @@ elseif(UNIX)
 endif()
 
 if(CMAKE_C_COMPILER_ID MATCHES "^(Intel)$")
-  set(CMAKE_C_FLAGS "-g -O3")
+  set(CMAKE_C_FLAGS "-g -O3" ${CMAKE_C_FLAGS})
```

Comment:
I think you want the `${CMAKE_C_FLAGS}` before and inside of the `" "` otherwise this is interpreted as a list.
The output will look like:
```
-g -O3;${CMAKE_C_FLAGS}
```
What we want is:
```
${CMAKE_C_FLAGS} -g -O3
```